ddp-workspace/projects/ddp-pancan/src/app/components/footer/footer.component.scss

```diff
@@ -1,5 +1,82 @@
 @import '../../../styles/variables.scss';
 
+$footer-background: #131314;
+$footer-divider-color: #444444;
```

Comment:
Suggest moving color variables into a separate file (`colors.scss` ?)

Reply:
I'm not sure about that. I checked mockups and I don't see we are going to use these colors anywhere else except the footer. I like the approach of minimizing the scope.

Reply:
ok, never mind